### PR TITLE
crate.service update to use crate/crate:testing

### PR DESCRIPTION
**Pull request by celaus**

### CODE REVIEW

**Comment by mikethebeer** [comment]
:tada: ack

**Comment by mikethebeer** [comment]
:tada: ack

File: play-crate-io/crate.service [hunk 1]
```diff
@@ -18,7 +18,7 @@ ExecStart=/bin/bash -c '\
         --volume /mnt/data1/crate:/data \
         --volume /tmp:/tmp \
         --env CRATE_HEAP_SIZE=16g \
-        crate:latest \
+        crate/crate:play \
```
Comment (chaudum) [inline play-crate-io/crate.service:21]: why do you use a new tag for that? we have a `:testing` tag already

Comment (mikethebeer) [inline play-crate-io/crate.service:21]: @chaudum `:testing` refers to our current testing release. on play.crate.io should only run versions that are getting stable on the same day.

**Pull request merged**